qemu/tests/multi_disk.py

```diff
@@ -246,28 +303,7 @@ def _do_post_cmd(session):
         err_msg = "Set disks num: %d" % stg_image_num
         err_msg += ", Get disks num in guest: %d"
         ostype = params["os_type"]
-        if ostype == "windows":
-            error_context.context("Get windows disk index that to "
-                                  "be formatted", test.log.info)
-            disks = utils_disk.get_windows_disks_index(session, stg_image_size, 300)
-            if len(disks) < stg_image_num:
-                test.fail("Fail to list all the volumes"
-                          ", %s" % err_msg % len(disks))
-            if len(disks) > drive_letters:
-                black_list.extend(utils_misc.get_winutils_vol(session))
-                disks = random.sample(disks, drive_letters - len(black_list))
-            error_context.context("Clear readonly for all disks and online "
-                                  "them in windows guest.", test.log.info)
-            if not utils_disk.update_windows_disk_attributes(session, disks):
-                test.fail("Failed to update windows disk attributes.")
-            dd_test = "no"
```

Comment:
Do you missed this line? And Could you please add the latest test results. Thanks.

Reply:
updated